logger_cb_test.go

```diff
@@ -0,0 +1,43 @@
+package libbpfgo
+
+import "testing"
+
+func TestLogFilterOutput(t *testing.T) {
+	tests := []struct {
+		libbpfPrintLevel int
+		output           string
+		expectedResult   bool
+	}{
+		{
+			output:         "libbpf: prog 'trace_check_map_func_compatibility': failed to create kprobe 'check_map_func_compatibility+0x0' perf event: No such file or directory\n",
```

Comment:
I believe we shouldn't filter anything internally by default. Consumers should choose their own filters, and since I assume most of these filters were hard coded for tracee, they should now be removed and set as filters from tracee itself.

So as for this test, we can test the same patterns but remove their hardcoding before. The test will first set the wanted pattern then test that they are correctly filtered.

Reply:
> I believe we shouldn't filter anything internally by default. Consumers should choose their own filters, and since I assume most of these filters were hard coded for tracee, they should now be removed and set as filters from tracee itself.

I agree.

> So as for this test, we can test the same patterns but remove their hardcoding before. The test will first set the wanted pattern then test that they are correctly filtered.

Ok. I'll tackle this soon. Thanks.